go/proto/ctrl/v1/deployment.proto

```diff
@@ -23,7 +23,7 @@ enum SourceType {
 }
 
 message CreateDeploymentRequest {
-  string workspace_id = 1;
```

Comment:
Can't we just get rid of that and shift everything by one? I believe you did for not to cause a breaking change, but that looks weird 😄.

Reply:
I didn't see this until now 😭 

https://github.com/unkeyed/unkey/pull/4056#discussion_r2399738568 I originally had it, but then it was giving me headaches to test against the cloud deployment vs local because the ctrl service would need to be merged and deployed, so I just followed best practices and reserved previously used proto assignments, even if its not in prod yet and wouldn't be breaking